src/Core/Commander/Providers/WMS_Provider.js

```diff
@@ -73,7 +73,6 @@ WMS_Provider.prototype.preprocessDataLayer = function preprocessDataLayer(layer)
     layer.style = layer.style || '';
     layer.transparent = layer.transparent || false;
     layer.bbox = layer.bbox || new BoundingBox();
-
```

Comment:
Unrelated change (it adds noise to the PR so I'd rather not have this change here)